client/lib/api/content/content_store.dart

```diff
@@ -100,39 +101,59 @@ abstract class _ContentStore with Store implements Updateable {
 
     // Note the logic enforcing that only *newer* versions replace older versions in
     // the *cache* must be enforced elsewhere.
-    await Future.wait([
-      updateBundle<IndexContent>(
-          'home_index', homeIndex, (cb) => IndexContent(cb), (v) {
-        homeIndex = v;
-      }),
-      updateBundle<AdviceContent>(
-          'travel_advice', travelAdvice, (cb) => AdviceContent(cb), (v) {
-        travelAdvice = v;
-      }),
-      updateBundle<FactContent>(
-          'get_the_facts', getTheFacts, (cb) => FactContent(cb), (v) {
-        getTheFacts = v;
-      }),
-      updateBundle<FactContent>(
-          'protect_yourself', protectYourself, (cb) => FactContent(cb), (v) {
-        protectYourself = v;
-      }),
-      updateBundle<IndexContent>(
-          'learn_index', learnIndex, (cb) => IndexContent(cb), (v) {
-        learnIndex = v;
-      }),
-      updateBundle<IndexContent>(
-          'news_index', newsIndex, (cb) => IndexContent(cb), (v) {
-        newsIndex = v;
-      }),
-      updateBundle<QuestionContent>('your_questions_answered',
-          questionsAnswered, (cb) => QuestionContent(cb), (v) {
-        questionsAnswered = v;
-      }),
-      updateBundle<PosterContent>(
-          'symptom_poster', symptomPoster, (cb) => PosterContent(cb), (v) {
-        symptomPoster = v;
-      })
-    ]);
+    try {
+      await Future.wait([
+        updateBundle<IndexContent>(
+            'home_index', homeIndex, (cb) => IndexContent(cb), (v) {
+          homeIndex = v;
+        }),
+        updateBundle<AdviceContent>(
+            'travel_advice', travelAdvice, (cb) => AdviceContent(cb), (v) {
+          travelAdvice = v;
+        }),
+        updateBundle<FactContent>(
+            'get_the_facts', getTheFacts, (cb) => FactContent(cb), (v) {
+          getTheFacts = v;
+        }),
+        updateBundle<FactContent>(
+            'protect_yourself', protectYourself, (cb) => FactContent(cb), (v) {
+          protectYourself = v;
+        }),
+        updateBundle<IndexContent>(
+            'learn_index', learnIndex, (cb) => IndexContent(cb), (v) {
+          learnIndex = v;
+        }),
+        updateBundle<IndexContent>(
+            'news_index', newsIndex, (cb) => IndexContent(cb), (v) {
+          newsIndex = v;
+        }),
+        updateBundle<QuestionContent>('your_questions_answered',
+            questionsAnswered, (cb) => QuestionContent(cb), (v) {
+          questionsAnswered = v;
+        }),
+        updateBundle<PosterContent>(
+            'symptom_poster', symptomPoster, (cb) => PosterContent(cb), (v) {
+          symptomPoster = v;
+        })
+      ]);
+
+      unawaited(
+        UserPreferences().setLastUpdatedContent(
+          DateTime.now().millisecondsSinceEpoch,
+        ),
+      );
+      print(
+        'ContentStore update finished',
+      );
+    } catch (e) {
+      unawaited(
+        FirebaseCrashlytics.instance.log(
```

Comment:
https://firebase.google.com/docs/crashlytics/customize-crash-reports?platform=android#log-excepts
```suggestion
        FirebaseCrashlytics.getInstance().recordException(e);
```